pvlib/iotools/psm3.py

```diff
@@ -296,12 +330,25 @@ def parse_psm3(fbuf):
     tz = 'Etc/GMT%+d' % -metadata['Time Zone']
     data.index = pd.DatetimeIndex(dtidx).tz_localize(tz)
 
+    if map_variables is None:
+        warnings.warn(
+            'PSM3 variable names will be renamed to pvlib conventions by '
+            'default starting in pvlib 0.11.0. Specify map_variables=True '
+            'to enable that behavior now, or specify map_variables=False '
+            'to hide this warning.', pvlibDeprecationWarning)
+        map_variables = False
+    if map_variables:
+        data = data.rename(columns=PSM3_VARIABLE_MAP)
+        metadata['latitude'] = metadata.pop('Latitude')
+        metadata['longitude'] = metadata.pop('Longitude')
+        metadata['elevation'] = metadata.pop('Elevation')
```

Comment:
If these are handled separately anyway, is there any point in listing them in the mapping dict?

Reply:
No, I suppose not.